ir/pattern/constraint.rs

```diff
@@ -2045,6 +2135,10 @@ impl<ID: IrID> FunctionCallBinding<ID> {
         self.ids_assigned().chain(self.function_call.argument_ids())
     }
 
+    pub fn ids_accepted(&self) -> impl Iterator<Item = ID> + '_ {
```

Comment:
why not just `argument_ids` ( you could rename `ids_assigned` to `assigne_ids` as well to make it match if you wanted, but not required imo)